common/app/common/jobs.scala

```diff
@@ -11,11 +12,25 @@ import Play.current
 object Jobs extends Logging {
   private val scheduler = StdSchedulerFactory.getDefaultScheduler()
   private val jobs = mutable.Map[String, () => Unit]()
+  private val outstanding = mutable.Map[String,AtomicInteger]().withDefaultValue(new AtomicInteger(0))
```

Comment:
One more thing, this Map is not thread safe. I tend to use the Java ConcurrentHashMap something like this http://stackoverflow.com/a/18667940